config/database.yml

```diff
@@ -1,26 +1,57 @@
-# MySQL. Versions 5.1.10 and up are supported.
```

Comment:
So as a practice we never commit the `database.yml` to the repo. This is because we don't dictate how each dev's DB setup is. 

Rather, push a file called `database.yml.sample` or something in the same style. This is for new devs to have a template of what to use.

Reply:
Okay got it po.